lib/ui.js

```diff
@@ -6,6 +6,20 @@ const scenario = require('./scenario');
 const ScenarioConfig = require('./interfaces/scenarioConfig');
 const FeatureConfig = require('./interfaces/featureConfig');
 const addDataContext = require('./data/context');
+const container = require('./container');
+
+
+const setContextTranslation = (context) => {
```

Comment:
what do think if we rename it to `passedContext`? It's kinda confusing when you see context and contexts. That's easy to mess them up :)

Reply:
Valid point! will do that :)